### ISSUE TITLE
Chunks, textures, inventory menu are gone

### ISSUE THREAD

**Issue by Tzipirigu** [MyM-ModpackTeam/GalacticScience#211]
Hi, I've been having this problem for a very long time. When I enter the map and want to open the inventory or change an object in the hotbar, the screen becomes a static color and changes as I move the mouse. Most textures do not exist, such as chests, UIs, chunks. I mention that I have a last generation laptop. I have a video I made now but I can't upload it because it's more than 10mb. Here are some photos to figure out the error.

![Error 1](https://github.com/MyM-ModpackTeam/GalacticScience/assets/91142910/84b248e6-2596-4ee1-a622-03e3abbc620a)
Chunks do not load in the house.

![Error 2](https://github.com/MyM-ModpackTeam/GalacticScience/assets/91142910/cb38f6b4-50bb-4c35-8d2c-0f88a509c7ae)
Chunks do not load outside the house.

![Error 3](https://github.com/MyM-ModpackTeam/GalacticScience/assets/91142910/f365c661-2790-4885-89bf-215ac50d9bb4)
The texture of the chests and stuff doesn't work

**Comment by ghost** [comment]
Are you by any chance using Optifine?

**Comment by Tzipirigu** [comment]
No. It is the first instance on this computer. Maybe it's installed in mods 🤔? I'll check right away and let you know!

**Comment by ghost** [comment]
Just so you don't need to wait for an answer, if you do use Optifine, disable multi-core rendering or try removing it completely. Also make sure your graphics card drivers are up to date.

**Comment by Stalin1984** [comment]
> No. It is the first instance on this computer. Maybe it's installed in mods 🤔? I'll check right away and let you know!

Hi, tell me please, have you been able to solve this problem? If so, how did you do it?

**Comment by BigGame2004** [comment]
Most likely some kind of unsupported gpu, can you give us some specs?

**Comment by Tzipirigu** [comment]
> have you been able to solve this problem?

No.
Holding a book, new world
![Holding a book, new world](https://github.com/user-attachments/assets/376b6231-d73a-4758-be07-e970f1f986f5)
Took a screenshot on the pause menu, I can't see anything!
![Took a screenshot on the pause menu, I can't see anything!](https://github.com/user-attachments/assets/a9371861-f088-442e-ba3f-26e10c9e893c)
Oxygen Tank in inventory, if I take it, it looks like it should
![Oxygen Tank in inventory](https://github.com/user-attachments/assets/ebe1834b-8cd0-47ae-a8c7-2cb2b33bebe9)

NO OPTIFINE, NEW PC, NEW INSTALLATION!

btw, I'm very sorry that I didn't write before. I didn't care anymore because of the error, and I thought it would be solved, but it still exists... at least not so bad.
I have a weird setting in video settings, anisotropic filtering. Maybe that has to do with the visual error?

my video settings
![my video settings](https://github.com/user-attachments/assets/3b34325e-a313-47f6-adba-f7867f7cf068)

**Comment by BigGame2004** [comment]
Does your computer have 2 gpus by any chance?

**Comment by Tzipirigu** [comment]
😐

**Comment by Tzipirigu** [comment]
https://github.com/MyM-ModpackTeam/GalacticScience/issues/209
Solved
